CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Changelog
 
-## 3.7.0-rc.0 / 2025-10-02
+## 3.7.0 / 2025-10-15
 
-* [CHANGE] Remote-write: Deprecate `prometheus_remote_storage_{samples,exemplars,histograms}_in_total` and `prometheus_remote_storage_highest_timestamp_in_seconds` metrics, see their respective descriptions for alternatives. #17065
+* [CHANGE] Remote-write: the following metrics are deprecated:
+   - `prometheus_remote_storage_samples_in_total`, use `prometheus_wal_watcher_records_read_total{type="samples"}` and `prometheus_remote_storage_samples_dropped_total` instead,
+   - `prometheus_remote_storage_histograms_in_total`, use `prometheus_wal_watcher_records_read_total{type=~".*histogram_samples"}` and `prometheus_remote_storage_histograms_dropped_total` instead,
+   - `prometheus_remote_storage_exemplars_in_total`, use `prometheus_wal_watcher_records_read_total{type="exemplars"}` and `prometheus_remote_storage_exemplars_dropped_total` instead,
+   - `prometheus_remote_storage_highest_timestamp_in_seconds`, use the more accurate `prometheus_remote_storage_queue_highest_timestamp_seconds` instead in dashboars and alerts to properly account for relabeling and better accuracy. #17065
```

Comment:
"dashboars" is cute but probably best to say "dashboards".
Also "more accurate" .. "better accuracy" ?